kernel/byterun/coq_interp.c

```diff
@@ -805,14 +816,15 @@ value coq_interprete
         /* creation des fonction cofix */
 
         p = sp;
-        size = nfunc + nvars + 2;
+        size = nfunc + nvars + 3;
         for (i=0; i < nfunc; i++) {
 
           Alloc_small(accu, size, Closure_tag);
           Code_val(accu) = pc+pc[i];
-          for (j = 0; j < nfunc; j++) Field(accu, j+1) = p[j];
+          Field(accu, 1) = Val_int(2);
+          for (j = 0; j < nfunc; ++j) Field(accu, j + 2) = p[j];
```

Comment:
Just a question of style. No idea why I changed it here. I suppose that, at some time, I had completely removed the line, and when I added it back later with `j+2`, I used my own style. I can change it to the style used elsewhere in the file. (That said, both styles are actually used in the file. But this might already have been my fault from an older commit.)

Reply:
No need to change it.

Reply:
No need to change the patch or the code as it was before? ;)

Reply:
Sorry, I mean that I do not care, so be lazy.